src/components/NewCommentForm.tsx

```diff
@@ -53,19 +53,23 @@ export const NewCommentForm: React.FC<Props> = ({ onSubmit }) => {
       body: !body,
     });
 
-    if (!name || !email || !body) {
+    if (!name || !email || !body || !post) {
       return;
```

Comment:
The form submission should not be blocked just because there is no selected post. Instead, an error message should be displayed to the user. Consider updating the condition.

Reply:
fixed